README.md

```diff
@@ -402,22 +475,64 @@ Example `process_exec` and `process_exit` events can be:
 </p>
 </details>
 
-For the rest of the use cases we will use the Tetragon CLI to give the output.
+#### Use case 2: Privileged Execution
+
+Tetragon also provides the ability to check process capabilities and kernel namespaces.
```

Comment:
"and kernel namespace access. This information would help us determine which process or Kubernetes pod has started with privileges or host namespace access that it should not have or gained privileges and host namespace access later in the lifecycle that it should not have. This would help us answer questions like "which Kubernetes pods are running with CAP_SYS_ADMIN in my cluster? which Kubernetes pods have host network or pid namespace access in my cluster?" 

Feel free to reword it, but it would be worth calling the exact use cases